app/src/main/res/values/strings.xml

```diff
@@ -447,6 +447,16 @@
     <string name="welcome_themes_title">Different themes</string>
     <string name="welcome_themes_message">Go to Settings, and you can choose from three type of themes : Black, Dark, White</string>
     <string name="welcome_merge_title">Merge PDF file</string>
+    <string name="welcome_text_to_pdf">Text to PDF</string>
```

Comment:
add strings for other locale files also. 
Other than it, looks awesome 👍

Reply:
@Swati4star I was thinking to make it a ` Beginner ` level issue.

Reply:
Cause moreover i don't know the translations in other languages.

Reply:
Add them in English only. 
Just the keys need to be present in every locale file.

Otherwise it causes issues while creating a release apk